### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with SVG Icon in Login/Signup Screen

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4765]
# Intro
This is an accessibility problem I found with the 'Show Password' button in the Login/Signup screens.

## Issue
The SVG icon in the button for making passwords visible has no `aria-label`. ***This is a violation of WCAG Success Criterion 4.1.2 (Level A) - Name, Role, Value***.  [See](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

## Description
***Success Criterion 4.1.2 (Level A) - Name, Role, Value*** requires that elements have accessible names and roles when they are perceivable to users or have other visual or auditory indications of their presence.

## Impact
Screen-readers would not work with it, for this reason. The output would be non-descriptive, with no indication of what the button does.

## Fix
Adding an aria-label. I will raise a PR for this shortly.
